src/integration/jackson/src/jvmMain/kotlin/community/flock/wirespec/integration/jackson/WirespecModule.kt

```diff
@@ -41,10 +41,16 @@ import java.io.IOException
  * @see Wirespec.Refined
  */
 class WirespecModule : SimpleModule() {
-    init {
+
+    override fun getModuleName(): String = "Wirespec Jackson Module"
+
+    override fun setupModule(context: SetupContext) {
         addSerializer(Wirespec.Refined::class.java, RefinedSerializer())
-        setDeserializerModifier(RefinedModifier())
+        addSerializer(Wirespec.Enum::class.java, EnumSerializer())
+        setDeserializerModifier(WirespecModifier())
+        super.setupModule(context)
```

Comment:
Moet je niet beginnen met de super call?

Reply:
When `super` is called first the modifier is nog registered. I reverted the to the Kotlin init sinds we dont need the context.